File: prefix-sum/1833-find-the-highest-altitude/1833-find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        maxAlt=0
        alt=0
        for i in range(len(gain)):
            alt += gain[i]
            if alt > maxAlt:
                maxAlt = alt
        return maxAlt
        

File: prefix-sum/1833-find-the-highest-altitude/1833-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxAlt = 0;
        int alt = 0;
        for (const int& g: gain){
            alt += g;
            if (alt > maxAlt) maxAlt = alt;
        }
        return maxAlt;
    }
};